applications/ParticleMechanicsApplication/custom_io/particle_vtk_output.cpp

```diff
@@ -0,0 +1,159 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Nicolò Crescenzio
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "particle_mechanics_application_variables.h"
+#include "particle_vtk_output.h"
+#include "includes/kratos_filesystem.h"
+
+namespace Kratos
+{
+
+Parameters VtkOutput::GetDefaultParameters()
+{
+    // IMPORTANT: when "output_control_type" is "time", then paraview will not be able to group them
+    Parameters default_parameters = Parameters(R"(
+    {
+        "model_part_name"                             : "PLEASE_SPECIFY_MODEL_PART_NAME",
+        "file_format"                                 : "binary",
+        "output_precision"                            : 7,
+        "output_control_type"                         : "step",
+        "output_interval"                             : 1.0,
+        "output_sub_model_parts"                      : false,
+        "output_path"                                 : "VTK_Output",
+        "custom_name_prefix"                          : "",
+        "custom_name_postfix"                         : "",
+        "save_output_files_in_folder"                 : true,
+        "entity_type"                                 : "automatic",
+        "write_ids"                                   : false,
+        "element_flags"                               : [],
+        "condition_flags"                             : [],
+        "gauss_point_variables_in_elements"           : []
+    })" );
+
+    return default_parameters;
+}
```

Comment:
do you need this method? Seems unused

Reply:
I'm using these parameters for the validation of the input settings in the constructor of the Python process. 
See the code here:

https://github.com/KratosMultiphysics/Kratos/blob/5cb788e0361ea7125df5c4a2a653dfce5afc679b/applications/ParticleMechanicsApplication/python_scripts/particle_vtk_output_process.py#L22-L23

I know that the validation of the input settings is already performed in the constructor of the C++ process, but if I am not wrong it always uses the default parameters of the base class `VtKOutput`. Now the problem is that the parameters of the base class `VtkOutput` contains some fields, such as `element_data_value_variables`, `condition_data_value_variables`, `nodal_data_value_variables`, `nodal_flags` and `nodal_solution_step_data_variables`, that must not be used when using the class `ParticleVtkOutput` because some errors arise (they basically have no meaning in this context). I just want to avoid an external user to copy the parameters from the base process and add some variables where they shouldn't be added.

I don't like too much the solution that I am proposing, but I failed to come up with a better idea. 
Do you have any suggestion?